src/compiler/builder.ts

```diff
@@ -1057,10 +1064,10 @@ function getBuildInfo(state: BuilderProgramState, bundle: BundleBuildInfo | unde
             outSignature: state.outSignature,
             latestChangedDtsFile,
             pendingEmit: !state.programEmitPending ?
-                undefined :  // Pending is undefined or None is encoded as undefined
+                undefined : // Pending is undefined or None is encoded as undefined
                 state.programEmitPending === getBuilderFileEmit(state.compilerOptions) ?
-                    false : // Pending emit is same as deteremined by compilerOptions
-                    state.programEmitPending, // Actual value
+                false : // Pending emit is same as deteremined by compilerOptions

```

Comment:
Doesnt this look wierd ?
ternary operator should add indent right ?

Reply:
In our config, it's supposed to be formatted kinda like a pattern match, but this ternary isn't formatted that way so its diff looks weird. I could potentially manually tweak it.